profiles/qtile.py

```diff
@@ -0,0 +1,55 @@
+# A desktop environment using "qtile" window manager with common packages.
+
+import archinstall
+
+is_top_level_profile = False
+
+# New way of defining packages for a profile, which is iterable and can be used out side
+# of the profile to get a list of "what packages will be installed".
+__packages__ = [
+	'qtile',
+	'alacritty',
+	'lightdm-gtk-greeter',
+	'lightdm',
+	'dmenu',
+	'pcmanfm',
+	'vim',
+	'nano',
+	'nitrogen',
+	
```

Comment:
delete blank spaces here for linter, and remove trailing comman from nitrogen. Also, is nitrogen really needed?

Reply:
For linter and the flake errors do I remove all blank spaces ?

Reply:
Just the blank lines where it's complaining.